Makefile

```diff
@@ -1,36 +1,47 @@
+# Heavily inspired by Lighthouse: https://github.com/sigp/lighthouse/blob/stable/Makefile
+# and Reth: https://github.com/paradigmxyz/reth/blob/main/Makefile
+.DEFAULT_GOAL := help
+
 VERSION := $(shell git describe --tags --always --dirty="-dev")
 
-.PHONY: all
-all: clean build
```

Comment:
Why remove the all target?
usually it is helpful to wrap clean & build in a single make command

Reply:
the all tarket is unused because the default is not printing the help